src/store/assets/asset.hooks.ts

```diff
@@ -56,3 +56,15 @@ export function useSearchInput() {
 export function useUpdateSearchInput() {
   return useUpdateAtom(searchInputStore);
 }
+
+export function useAssetByContractId(contractId: string) {
+  if (!contractId) return
```

Comment:
This breaks rules of hooks too. Plus, we should be sure it isn't undefined, as it's not typed as so.